pkg/plugins/document/plugin.go

```diff
@@ -85,3 +88,9 @@ func (p *UnimplementedDocumentPlugin) Delete(key *Key) error {
 func (p *UnimplementedDocumentPlugin) Query(collection *Collection, expressions []QueryExpression, limit int, pagingToken map[string]string) (*QueryResult, error) {
 	return nil, fmt.Errorf("UNIMPLEMENTED")
 }
+
+func (p *UnimplementedDocumentPlugin) QueryStream(collection *Collection, expressions []QueryExpression, limit int) DocumentIterator {
+	return func() (*Document, error) {
+		return nil, fmt.Errorf("UNIMPLEMENTED")
+	}
```

Comment:
At some stage we should update all the plugin with a GRPC code for unimplemented:

```
return status.Errorf(codes.Unimplemented, "...")
```

Reply:
yup, lets raise an issue for this to ensure its done consistently across the board.